app/api/entities/team.rb

```diff
@@ -3,9 +3,16 @@ class Entities::Team < Entities::BaseEntity
   expose :name
   expose :logo
   expose :disable_registration
+  expose :admins
 
   with_options(format_with: :iso_timestamp) do
     expose :created_at, if: lambda { |i, o| i.created_at.present? }
     expose :archived_at, if: lambda { |i, o| i.archived_at.present? }
   end
+
+  private
+
+  def admins
+    object.enrollments.includes(:user).where(is_admin: true).map(&:user) rescue []
```

Comment:
Using this forces you to includes `:enrollments`in Teams query. Make sure you are adding it in necessary places.